### PR TITLE
Telegram / sell duration

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
Show the duration of the trade together with the sell notifiction.

Trade-duration is something very interresting ... (did the bot make 2% in 1 day? in 1 hour? ...)

## Quick changelog

- Add Duration to sell-notification

### CODE REVIEW

File: freqtrade/rpc/telegram.py [hunk 1]
```diff
@@ -144,13 +144,16 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
...
+            msg['duration'] = msg['close_date'].replace(
+                microsecond=0) - msg['open_date'].replace(microsecond=0)
 
             message = ("*{exchange}:* Selling {pair}\n"
                        "*Rate:* `{limit:.8f}`\n"
                        "*Amount:* `{amount:.8f}`\n"
                        "*Open Rate:* `{open_rate:.8f}`\n"
                        "*Current Rate:* `{current_rate:.8f}`\n"
                        "*Sell Reason:* `{sell_reason}`\n"
+                       "*Duration:* `{duration}`\n"
```
Comment (hroff-1902) [inline freqtrade/rpc/telegram.py]: Duration may be included in two forms: as a human-readable 5h 33m 50.3s and in minutes (1350.3 min), separated somehow, like "5h 33m 50.3s / 1350.3 min" or "5h 33m 50.3s (1350.3 min)"

**Comment by hroff-1902** [comment]
I'm going (soon) to change all "mins" in the code into "min", since that's the correct usage of abbreviation for minutes (see SI standard, "2 kg" not "2 kgs", "60 min", not "60 mins", etc...)
So you can either fix this here or leave it as it is for now, which better suites you...
Anything else is LGTM 👍 , approving.

**Comment by xmatthias** [comment]
> People frequently use "mins" and "hrs", it's not quite correct, however...)

you got to be kidding me ... 
i've even had it with "min" initially but it "felt" wrong ... so i was certain you'd point out THAT as being wrong - that's why i added the "plurals" in the first place ... :laughing:

**Comment by hroff-1902** [comment]
https://english.stackexchange.com/a/154441

> The SI unit for time is Seconds (s). Hours (h) and minutes (min) are accepted to be used within the SI even if not standardised. In scientific writing you should probably stick to those abbreviations (note that it's 60 min not 60 mins).

**Pull request merged**